### PR TITLE
[TW-1342] Flows for basic users

### PR DESCRIPTION
**Pull request by petewitcher-postman**
This is a one-liner capturing how flows is now an add-on in the Basic plan.

### CODE REVIEW

File: src/pages/docs/postman-flows/gs/flows-overview.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Check out these Flows Workspaces filled with examples:
 
 These workspaces include example flows utilizing a multitude of APIs for different products and tools, plus all the related collections and environments needed. You can [fork](/docs/collaborating-in-postman/using-version-control/forking-entities/) the flows to your own workspace, try them out, and then make your own changes and customizations.
 
-> Postman Flows is free to try on the Free plan for up to three users. Postman Flows is available as an add-on to the annual [Professional plan](https://www.postman.com/pricing/). Postman Flows isn't currently available in the Basic and Enterprise plans.
+> Postman Flows is free to try on the Free plan for up to three users. Postman Flows is available as an add-on to the annual [Basic and Professional plans](https://www.postman.com/pricing/). Postman Flows isn't currently available in the Enterprise plan.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/gs/flows-overview.md:64]: I'm sure this is fine, but I just wanted to double check. Does this change impact the plans we mention in the context of the Flow Editor role in the following sections?

* [Enabling Postman Flows](https://learning.postman.com/docs/postman-flows/gs/flows-overview/#enabling-postman-flows)
* [Team roles](https://learning.postman.com/docs/collaborating-in-postman/roles-and-permissions/#team-roles)

**Review by nerminamiller-postman — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM, but make sure to address Matthew's question

File: src/pages/docs/postman-flows/gs/flows-overview.md (hunk 1 above)
Comment (petewitcher-postman) [reply src/pages/docs/postman-flows/gs/flows-overview.md:64]: Great catch, thanks Matthew. Confirmed with Labs the Flow Editor role applies to Basic and Professional plans, and Flows will be available in monthly and annual plans for both as of September 15. Updated both files accordingly.

**Pull request merged**